src/jobs/deploy-eb.yml

```diff
@@ -1,9 +1,88 @@
 description: >
-  This job deploys an application to aws elasticbeanstalk.
+  This job deploys an application to aws elasticbeanstalk. Deploy-Eb requires publish-docker
 
 executor: node/default
 
+parameters:
+  # Required Parameters
+  application-name:
+    type: string
+    description: |
+      The name of the elastic beanstalk instance that will be updated in this job
+  environment-name:
+    type: string
+    description: |
+      The name of the elastic beanstalk instance that was given during eb init
+  repo-name:
```

Comment:
this is already stored in circleci, $CIRCLE_PROJECT_REPONAME
see: https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables